.dwollaci.yml

```diff
@@ -26,7 +26,7 @@ stages:
         . ${NVM_DIR}/nvm.sh --no-use
         nvm install
         npm install -g npm
-        npm install -g serverless
+        npm install -g serverless@"<4.0.0"
```

Comment:
remind me why we have to constrain to <v4? cost?

Reply:
There was an update to the license in v4 that was causing use to get an error during the deploy. We would need to figure out how that impacts our deploy process and any other changes required before migrating to v4.